app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -164,26 +179,86 @@ const FormOverview = ({
       </Form.Group>
 
       {experiment && (
-        <Form.Group controlId="publicDescription">
-          <Form.Label className="d-flex align-items-center">
-            Public description
-            {isMissingField!("public_description") && (
-              <InlineErrorIcon
-                name="description"
-                message="Public description cannot be blank"
+        <>
+          <Form.Group controlId="publicDescription">
+            <Form.Label className="d-flex align-items-center">
+              Public description
+              {isMissingField!("public_description") && (
+                <InlineErrorIcon
+                  name="description"
+                  message="Public description cannot be blank"
+                />
+              )}
+            </Form.Label>
+            <Form.Control
+              as="textarea"
+              rows={3}
+              {...formControlAttrs("publicDescription", {})}
+            />
+            <Form.Text className="text-muted">
+              This description will be public to users on about:studies
+            </Form.Text>
+            <FormErrors name="publicDescription" />
+          </Form.Group>
+
+          <Form.Group controlId="documentationLinks">
```

Comment:
Since this bit of the form is getting decently chonky, it might be handy to split it out into its own component. But, since add/remove is headed for a follow-up, that'd be a decent place to do it

Reply:
For sure agree. I'll split it out in the add/remove work!